test/test_scenario.py

```diff
@@ -0,0 +1,89 @@
+from src.hisp.helpers import Scenario
+import os
+import pytest
+import numpy as np
+import matplotlib.pyplot as plt
+
+current_dir = os.path.dirname(__file__)
+scenario_path = os.path.join(current_dir, "scenario_test.txt")
```

Comment:
💡 Instead of making text files manually, we should look into adding the possibility to create a scenario in python (and export it to txt)

Example interface:

```python
from hisp import Scenario

my_scenario = Scenario()
my_scenario.add_pulse(pulse_type="FP", ramp_up=..., flattop=..., nb_pulses=...)
my_scenario.add_pulse(pulse_type="FP", ramp_up=..., flattop=..., nb_pulses=...)

my_scenario.export_to_txt("scenario.txt")
```

This would greatly help testing and reduce manual steps